plasmapy/diagnostics/thomson.py

```diff
@@ -111,12 +129,39 @@ def spectral_density(
 
     electron_vel : `~astropy.units.Quantity`, shape (Ne, 3), optional
         Velocity of each electron component in the rest frame. (convertible to m/s)
+        If set, overrides electron_vdir and electron_speed.
         Defaults to a stationary plasma [0, 0, 0] m/s.
 
+    electron_vdir : np.ndarray, shape (Ne,3), optional
+        Unit vectors describing the velocity of each electron population.
+        Unit vectors will be normalized, but must not contain all zero elements.
+        Setting electron_vel overrides this keyword.
+
+    electron_speed : `~astropy.units.Quantity`, shape (Ne), optional
+        A scalar speed for each electron population. Must be used along with
+        electron_vdir. The electron_vel is calculated from these keywords as
+
+        electron_vel = electron_speed * electron_vdir
```

Comment:
```suggestion
        ``electron_vel = electron_speed * electron_vdir``
```

When you're referring to code, you want to make sure to surround the code in double back ticks ` `` `.  This will make sure the proper code styling is added in the documentation.  Use the single back tick `` ` `` when you're trying to reference documentation for a function, class, module, etc.

I'm just highlighting the case here, but there are several places in the docstrings where this needs to be applied.

Reply:
Ok, I think I've changed all of them.